frontend/src/metabase/metadata/components/CoercionStrategyPicker/utils.unit.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { humanizeCoercionStrategy } from "./utils";
+
+describe("humanizeCoercionStrategy", () => {
```

Comment:
This file was moved, but I wrapped everything in this `describe`, since I renamed this file from `humanizeCoercionStrategy.unit.spec.ts` to `utils.unit.spec.ts`. No other changes.